src/app/static/src/app/components/forms/DynamicFormMultiSelect.vue

```diff
@@ -1,37 +1,42 @@
 <template>
-    <div :class="cssClass">
+    <div>
         <tree-select :multiple="true"
                      :clearable="false"
                      v-model="value"
                      :options="formControl.options"></tree-select>
-        <input type="hidden" :value="value" :name="formControl.name"/>
+        <input type="hidden" :value="formControl.value" :name="formControl.name"/>
```

Comment:
Tbh it's not needed now. Previously it was because the html form was the source of truth for the data, so we needed to keep track of the value in an input. I left it in because I thought maybe it's nice that you could still inspect the form element and see all the values there if you wanted to, or indeed create a FormData object from it.